src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -205,7 +238,7 @@ export const SimplePowerBIReportEmbed = ({
     let content;
     try {
       content = (
-        <PowerBIEmbed cssClassName={classes.report} embedConfig={embedConfig} getEmbeddedComponent={setReport} />
+        <PowerBIEmbed cssClassName={classes.report} embedConfig={embedConfig} getEmbeddedComponent={() => setReport} />
```

Comment:
I think this change will break our usage of the `getEmbeddedComponent` prop. The previous version 
```
getEmbeddedComponent={setReport}
```
was equivalent to 
```
getEmbeddedComponent={(embedObject) => setReport(embedObject)}
```
it had the [expected function signature](https://github.com/microsoft/powerbi-client-react/blob/master/README.md#props-interface) `(embeddedComponent: Embed): void`
Yet the signature of the new version is quite different: `() => func`

Reply:
oh, thank you for the remark, I didn't think about it. Now, we can rollback this change as it was necessary to avoid the app crash because of `<StrictMode/> ` that we decided to disable :wink: